merge.go

```diff
@@ -139,7 +139,23 @@ func Merge(memGraph *lpg.Graph, dbGraph *DBGraph, config Config) ([]Delta, error
 				}
 			}
 		}
-		deltas = mergeSubtree(n, foundDBEntity, dbGraph, nodeAssociations, edgeAssociations, deltas)
+		sfx := strings.LastIndex(schemaPV.AsString(), "/")
```

Comment:
ls.FilterNonlayerTypes(node.GetLabels()), iterate through those and check if there is a config for it.